### DISCUSSION TITLE
Help me to make it work with windows please

### DISCUSSION THREAD

**Discussion by thegreat7man** [NaitLee/Cat-Printer#92] [Q&A]
hi all,I can't find any archives with windows version. I tried to download main archive and start .bat but nothing happened. I've installed python and have new windows 10

**Comment by NaitLee** [comment]
For ease of life you may want to use Kitty Printer instead, which is a simple Web app: https://print.unseen-site.fun/
(its source code is at https://github.com/NaitLee/kitty-printer)

Nowadays Cat Printer is primarily for devs to hack with automation.

**Reply by thegreat7man** [reply]
thank you for respond! can I make it work offline ?

**Reply by NaitLee** [reply]
Kitty Printer is still the easist to get to work locally:
- install [Deno](https://deno.com/)
- clone [kitty-printer repository](https://github.com/NaitLee/kitty-printer), or simply download its source code
- inside the source code directory, run the command `deno task start` to have a local service you can use offline

I will come back later with how to get Cat Printer to work. It became complicated nowadays.

**Reply by NaitLee** [reply]
To get Cat Printer to work:
- Install python & pip, clone this repository with git
- Install `bleak` with this command: `pip install bleak`
- After that, in the source code directory, execute `py server.py` to have the web interface, or `py printer.py` for a command line interface.
- In case you have installed python from microsoft store, the `pip` is automatically installed, and you need to use `python` instead of `py`.
- Probably python already have associated `.py` files. You can just double-click on `server.py` to have a web interface.

(Thankfully windows isn't affected by newest pip behavior that requires a virtual environment...)

**Reply by thegreat7man** [reply]
thank you so much
